plugins/package-managers/pub/src/main/kotlin/model/Lockfile.kt

```diff
@@ -72,6 +72,9 @@ internal data class PackageInfo(
     )
 }
 
+internal val PackageInfo.isProject: Boolean
```

Comment:
I believe this should be placed into `pub.kt` so that the `Lockfile.kt` reflects solely the 3rd party stuff.